README.md

```diff
@@ -16,10 +16,12 @@ For images, you must host them somewhere, eg Github, Imgur or I can put them on
 
 ## Getting started
 
-- Download repo
-- cd `/repo/`
-- `yarn`
-- Start it: `electron .`
+```sh
+git clone 'git@github.com:xivapi/ffxiv-launcher.git' # Clone the repo!
+cd ffxiv-launcher
+yarn install        # Snag the dependencies!
+yarn start          # Build and run it locally in dev mode!
```

Comment:
Does this process also compile the css/js, with them being `gitignore`, this needs to be a step.